lib/node_modules/@stdlib/blas/base/gaxpy/lib/ndarray.js

```diff
@@ -49,11 +55,21 @@ var M = 4;
 function gaxpy( N, alpha, x, strideX, offsetX, y, strideY, offsetY ) {
 	var ix;
 	var iy;
+	var ox;
+	var oy;
 	var m;
 	var i;
 	if ( N <= 0 || alpha === 0.0 ) {
 		return y;
 	}
+	ox = arraylike2object( x );
+	oy = arraylike2object( y );
+	if ( ox.accessorProtocol || oy.accessorProtocol ) {
+		accessors( N, ox, strideX, offsetX, oy, strideY, offsetY );
+		return oy.data;
+	}
+	ix = offsetX;
+	iy = offsetY;
```

Comment:
```suggestion
```

Reply:
You already have offset assignment at L73/L74. This is clearly a copy-paste mistake.